build.gradle

```diff
@@ -704,6 +704,7 @@ uiTestTargets.each { version ->
 
     if (project.hasProperty('headless')) {
       environment 'IDEPROBE_DISPLAY', 'xvfb'
+      environment 'IDEPROBE_SCREENSHOTS_DIR', '/thread-dump/dir'
```

Comment:
I think you can skip this one completely... as per `org.virtuslab.ideprobe.ide.intellij.InstalledIntelliJ#startProcess`, this will be set up to `/tmp/ide-probe/screenshots/` by default 🤔

Reply:
it is a little bit strange ;/

Reply:
❓

Reply:
mean that we cannot specify our own screenshots directory - it must be always /tmp/ide-probe/screenshots (+ additional directories created from IDEPROBE_TEST_SUITE and IDEPROBE_TEST_CASE)

Reply:
Okay... and what is the problem with uploading artifacts from that default `/tmp/ide-probe/screenshots` directory? 🤔

Reply:
(btw, pls open an issue against ide-probe if `IDEPROBE_SCREENSHOTS_DIR` doesn't work as expected indeed!)

Reply:
Yeah... but still, worth checking why `IDEPROBE_SCREENSHOTS_DIR` (although this can be done separately not to overextend this task). Actually... I have a _niesmiały pomysł_ that you might contribute a bit to ide-probe (if you like) as well, as this is a full-fledged Scala project (as opposed to this one, which is only half-assed Scala ;p)

Reply:
maybe a good idea :> and also discovered that without declaring IDEPROBE_TEST_SUITE and IDEPROBE_TEST_CASE pictures are not taking at all (from ide-probe code it should be taken from some environments but it doesn't seem to work). When I finish this task I will collect these cases together with some CI examples